src/Raven.Client/Documents/Session/Operations/GetRevisionOperation.cs

```diff
@@ -46,7 +45,12 @@ public GetRevisionOperation(InMemoryDocumentSessionOperations session, IEnumerab
 
         public GetRevisionsCommand CreateRequest()
         {
-            _session.IncrementRequestCount();
+            if (_command.ChangeVectors is not null)
+                return _session.CheckIfChangeVectorAlreadyIncluded(_command.ChangeVectors) ? null : _command ;
+            
+            if (_command.ChangeVector is not null)
```

Comment:
duplicated code?

Reply:
nope, It is because there are two commands one for change vector and one for change vectors